starter-code/src/viking.js

```diff
@@ -1,11 +1,51 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+  attack() {
+    return this.strength;
+  }
+  receiveDamage(damage) {
+    this.health = this.health - damage;
+  }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength) {
+    super(health, strength);
+    this.name = name;
+  }
 
-// Saxon
-class Saxon {}
+  receiveDamage(damage) {
+    this.health = this.health - damage;
```

Comment:
if you call 
`super.receiveDamage(damage)`
It will call the parents method so you don't have to rewrite the logic : )